README.md

```diff
@@ -1,6 +1,63 @@
-# SLSA Enhancement Proposals (SEPs)
+# SLSA Proposal Documents
 
-This repository contains proposals for enhancing [SLSA](https://slsa.dev).
+This repository contains "proposal documents" that supplement larger changes to
+[SLSA].
 
-**TODO:** Document the process. In the meantime, see
-[issue #296](https://github.com/slsa-framework/slsa/issues/296).
+**Proposals represent a snapshot in time.** They are intended to explain a
+proposed change and its rationale. They are **not** intended to serve as
+long-term documentation once the change is implemented. Long-term documentation
+should instead live in the [main SLSA repo].
+
+## Proposal index
+
+Number                 | Title                        | [Status]
+---------------------- | ---------------------------- | ---------
+[0001](0001/README.md) | Creation of proposal process | IN REVIEW
+
+## How to create a proposal document
+
+See [CONTRIBUTING] for a description of the process. Proposal documents are not
+needed for all changes.
+
+1.  Find or create a relevant [GitHub Issue] in the main SLSA repo to discuss
+    this idea.
+2.  Fork this repository and create a PR containing the new document:
+    *   Create `####/index.md` using [0001/README.md](0001/README.md) as a
```

Comment:
```suggestion
    *   Create `####/README.md` using [0001/README.md](0001/README.md) as a
```
I assume we want the proposal to README.md, like proposal 0001, not index.md?

Reply:
Oops, typo, thanks for catching. (Forgot to revert from an earlier GH Pages-based version.)